### ISSUE TITLE
[jitstress] nullref in DefaultMode_Missing_AllMethods_ReportsDiagnostic

### ISSUE THREAD

**Issue by markples** [dotnet/runtime#77153]
runtime-coreclr libraries-jitstress-random
Libraries Test Run checked coreclr Linux arm64 Release

https://dev.azure.com/dnceng-public/public/_build/results?buildId=53798&view=logs&j=90b83595-294f-5012-aa73-2e7af0be6394&t=51866677-f9ae-5538-b7aa-4c8484d0a9b9
https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-e7b4a61233f741c9a1/LibraryImportGenerator.Unit.Tests/1/console.7353caab.log?helixlogtype=result

```
+ printenv
+ grep COMPlus
COMPlus_TieredCompilation=0
COMPlus_JitStress=274
COMPlus_DbgMiniDumpName=/home/helixbot/dotnetbuild/dumps/coredump.%d.dmp
COMPlus_DbgEnableMiniDump=1
...
    LibraryImportGenerator.UnitTests.CustomMarshallerAttributeAnalyzerTests_StatefulValueShapeValidation.DefaultMode_Missing_AllMethods_ReportsDiagnostic [FAIL]
      System.AggregateException : One or more errors occurred. (Object reference not set to an instance of an object.)
      ---- System.NullReferenceException : Object reference not set to an instance of an object.
      Stack Trace:
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/TaskReplicator.cs(155,0): at System.Threading.Tasks.TaskReplicator.Run[TState](ReplicatableUserAction`1 action, ParallelOptions options, Boolean stopOnFirstFailure)
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(994,0): at System.Threading.Tasks.Parallel.ForWorker[TLocal](Int32 fromInclusive, Int32 toExclusive, ParallelOptions parallelOptions, Action`1 body, Action`2 bodyWithState, Func`4 bodyWithLocal, Func`1 localInit, Action`1 localFinally)
        --- End of stack trace from previous location ---
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(3121,0): at System.Threading.Tasks.Parallel.ThrowSingleCancellationExceptionOrOtherException(ICollection exceptions, CancellationToken cancelToken, Exception otherException)
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(1140,0): at System.Threading.Tasks.Parallel.ForWorker[TLocal](Int32 fromInclusive, Int32 toExclusive, ParallelOptions parallelOptions, Action`1 body, Action`2 bodyWithState, Func`4 bodyWithLocal, Func`1 localInit, Action`1 localFinally)
           at Roslyn.Utilities.RoslynParallel.For(Int32 fromInclusive, Int32 toExclusive, Action`1 body, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceNamespaceSymbol.ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceModuleSymbol.ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceAssemblySymbol.ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetSourceDeclarationDiagnostics(SyntaxTree syntaxTree, Nullable`1 filterSpanWithinTree, Func`4 locationFilterOpt, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetDiagnosticsWithoutFiltering(CompilationStage stage, Boolean includeEarlierStages, BindingDiagnosticBag builder, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetDiagnostics(CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.Diagnostics.CompilationWithAnalyzers.GetAllDiagnosticsWithoutStateTrackingAsync(ImmutableArray`1 analyzers, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.Diagnostics.CompilationWithAnalyzers.GetAllDiagnosticsAsync(CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs(926,0): at Microsoft.CodeAnalysis.Testing.AnalyzerTest`1.GetSortedDiagnosticsAsync(Solution solution, ImmutableArray`1 analyzers, ImmutableArray`1 additionalDiagnostics, CompilerDiagnostics compilerDiagnostics, IVerifier verifier, CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(734,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.FixAllAnalyerDiagnosticsInScopeAsync(FixAllScope scope, ImmutableArray`1 analyzers, ImmutableArray`1 codeFixProviders, Nullable`1 codeFixIndex, String codeFixEquivalenceKey, Action`2 codeActionVerifier, Project project, Int32 numberOfIterations, IVerifier verifier, CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(503,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.VerifyFixAsync(String language, ImmutableArray`1 analyzers, ImmutableArray`1 codeFixProviders, SolutionState oldState, SolutionState newState, Int32 numberOfIterations, Func`10 getFixedProject, IVerifier verifier, CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(472,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.VerifyFixAsync(SolutionState testState, SolutionState fixedState, SolutionState batchFixedState, IVerifier verifier, CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(309,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.RunImplAsync(CancellationToken cancellationToken)
        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs(181,0): at Microsoft.CodeAnalysis.Testing.AnalyzerTest`1.RunAsync(CancellationToken cancellationToken)
        /_/src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CustomMarshallerAttributeFixerTest.cs(140,0): at LibraryImportGenerator.UnitTests.CustomMarshallerAttributeFixerTest.VerifyCodeFixAsync(String source, String fixedSource, DiagnosticResult[] diagnostics)
        /_/src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CustomMarshallerAttributeFixerTests_StatefulValueShapeValidation.cs(342,0): at LibraryImportGenerator.UnitTests.CustomMarshallerAttributeAnalyzerTests_StatefulValueShapeValidation.DefaultMode_Missing_AllMethods_ReportsDiagnostic()
        --- End of stack trace from previous location ---
        ----- Inner Stack Trace -----
           at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetUnaliasedReferencedAssemblies(ArrayBuilder`1 assemblies)
           at Microsoft.CodeAnalysis.CSharp.Symbols.AssemblySymbol.GetTopLevelTypeByMetadataName(MetadataTypeName& metadataName, AssemblyIdentity assemblyOpt, Boolean includeReferences, Boolean isWellKnownType, ValueTuple`2& conflicts, DiagnosticBag warnings, Boolean ignoreCorLibraryDuplicatedTypes)
           at Microsoft.CodeAnalysis.CSharp.Symbols.AssemblySymbol.GetTypeByMetadataName(String metadataName, Boolean includeReferences, Boolean isWellKnownType, ValueTuple`2& conflicts, Boolean useCLSCompliantNameArityEncoding, DiagnosticBag warnings, Boolean ignoreCorLibraryDuplicatedTypes)
           at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetWellKnownType(WellKnownType type)
           at Microsoft.CodeAnalysis.CSharp.Binder.CheckAttributeTypeViability(Symbol symbol, Boolean diagnose, DiagnosticInfo& diagInfo, CompoundUseSiteInfo`1& attributeTypeViabilityUseSiteInfo)
           at Microsoft.CodeAnalysis.CSharp.Binder.LookupAttributeType(LookupResult result, NamespaceOrTypeSymbol qualifierOpt, String name, Int32 arity, ConsList`1 basesBeingResolved, LookupOptions options, Boolean diagnose, CompoundUseSiteInfo`1& useSiteInfo)
           at Microsoft.CodeAnalysis.CSharp.Binder.BindNonGenericSimpleNamespaceOrTypeOrAliasSymbol(IdentifierNameSyntax node, BindingDiagnosticBag diagnostics, ConsList`1 basesBeingResolved, Boolean suppressUseSiteDiagnostics, NamespaceOrTypeSymbol qualifierOpt)
           at Microsoft.CodeAnalysis.CSharp.Binder.BindNamespaceOrTypeOrAliasSymbol(ExpressionSyntax syntax, BindingDiagnosticBag diagnostics, ConsList`1 basesBeingResolved, Boolean suppressUseSiteDiagnostics)
           at Microsoft.CodeAnalysis.CSharp.Binder.BindTypeOrAlias(ExpressionSyntax syntax, BindingDiagnosticBag diagnostics, ConsList`1 basesBeingResolved, Boolean suppressUseSiteDiagnostics)
           at Microsoft.CodeAnalysis.CSharp.Binder.BindType(ExpressionSyntax syntax, BindingDiagnosticBag diagnostics, ConsList`1 basesBeingResolved, Boolean suppressUseSiteDiagnostics)
           at Microsoft.CodeAnalysis.CSharp.Binder.BindAttributeTypes(ImmutableArray`1 binders, ImmutableArray`1 attributesToBind, Symbol ownerSymbol, NamedTypeSymbol[] boundAttributeTypes, Action`1 beforeAttributePartBound, Action`1 afterAttributePartBound, BindingDiagnosticBag diagnostics)
           at Microsoft.CodeAnalysis.CSharp.Symbol.LoadAndValidateAttributes(OneOrMany`1 attributesSyntaxLists, CustomAttributesBag`1& lazyCustomAttributesBag, AttributeLocation symbolPart, Boolean earlyDecodingOnly, Binder binderOpt, Func`2 attributeMatchesOpt, Action`1 beforeAttributePartBound, Action`1 afterAttributePartBound)
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceNamedTypeSymbol.GetAttributesBag()
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceNamedTypeSymbol.GetAttributes()
           at Microsoft.CodeAnalysis.CSharp.Symbols.SourceMemberContainerTypeSymbol.ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
           at Microsoft.CodeAnalysis.CSharp.Symbol.ForceCompleteMemberByLocation(SourceLocation locationOpt, Symbol member, CancellationToken cancellationToken)
           at Roslyn.Utilities.RoslynParallel.<>c__DisplayClass1_0.<For>g__errorHandlingBody|0(Int32 i)
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(1062,0): at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`1.<ForWorker>b__1(RangeWorker& currentWorker, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
        --- End of stack trace from previous location ---
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(1105,0): at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`1.<ForWorker>b__1(RangeWorker& currentWorker, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/TaskReplicator.cs(77,0): at System.Threading.Tasks.TaskReplicator.Replica.Execute()
```

cc @dotnet/jit-contrib

**Comment by AndyAyersMS** [comment]
There wasn't a core dump generated. cc @JulieLeeMSFT @hoyosjs in case this is of interest.
```
Looking around for any Linux dump..
... found no dump in /root/helix/work/workitem/e
```
I have been having trouble reproing this on my Volterra (either with local build or runfo download artifacts), though just now I did get a similar looking error. Let me see if I can get this to happen with any frequency.
```
=== TEST EXECUTION SUMMARY ===
   LibraryImportGenerator.Unit.Tests  Total: 653, Errors: 0, Failed: 0, Skipped: 1, Time: 66.310s
Microsoft.DotNet.XUnitConsoleRunner v2.5.0 (64-bit .NET 8.0.0-ci)
  Discovering: LibraryImportGenerator.Unit.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  LibraryImportGenerator.Unit.Tests (found 164 test cases)
  Starting:    LibraryImportGenerator.Unit.Tests (parallel test collections = on, max threads = 4)
    LibraryImportGenerator.UnitTests.Compiles.ValidateSnippetsWithMarshalType [SKIP]
      No current scenarios to test.
    LibraryImportGenerator.UnitTests.Compiles.ValidateSnippets(id: "Compiles.cs:367", source: "\nusing System.Runtime.InteropServices;\nusing Sys"...) [FAIL]
      System.AggregateException : One or more errors occurred. (Object reference not set to an instance of an object.)
      ---- System.NullReferenceException : Object reference not set to an instance of an object.
      Stack Trace:
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/TaskReplicator.cs(155,0): at System.Threading.Tasks.TaskReplicator.Run[TState](ReplicatableUserAction`1 action, ParallelOptions options, Boolean stopOnFirstFailure)
        /_/src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs(994,0): at System.Threading.Tasks.Parallel.ForWorker[TLocal](Int32 fromInclusive, Int32 toExclusive, ParallelOptions parallelOptions, Action`1 body, Action`2 bodyWithState, Func`4 bodyWithLocal, Func`1 localInit, Action`1 localFinally)
        --- End of stack trace from previous location ---
```

**Comment by hoyosjs** [comment]
I'll need to check, but this feels like an in-xunit failure. The test harness essentially just swallows the exception, and nothing grabs a dump. @ViktorHofer do you know anything we have other than something like the remote executor that can help here? A quick thing to help could be wrapping in a try catch and have the catch use an IPC command to dump the test process if supported, but that's a feature. I can't find anything that xUnit itself supports.

**Comment by AndyAyersMS** [comment]
Yes, it probably is a failure in xunit -- of course the from the jit standpoint (especially in stress) its never clear which methods are the test cases and which methods are the test harness.

Looks like I can repro maybe 10% of the time, so should be able to make some slow headway pinning this one down.

**Comment by AndyAyersMS** [comment]
> Looks like I can repro maybe 10% of the time, so should be able to make some slow headway pinning this one down.

No luck so far trying stress range reduction. May need to rethink my approach.

**Comment by ViktorHofer** [comment]
> I'll need to check, but this feels like an in-xunit failure. The test harness essentially just swallows the exception, and nothing grabs a dump. @ViktorHofer do you know anything we have other than something like the remote executor that can help here?

In case it helps, the Microsoft.DotNet.XUnitConsoleRunner mentioned in the log is our own fork that lives in arcade. You can easily change it to your needs. https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.XUnitConsoleRunner

**Comment by AndyAyersMS** [comment]
This test passed in the most recent set of runs -- note each run uses a different stress key. So the issue is not pervasive, and I think we can remove the blocking tag.

**Comment by AndyAyersMS** [comment]
Seems possible this is a similar issue to #77661? Am going to try collecting and then replaying with and without the fix to see if I see diffs.

**Comment by AndyAyersMS** [comment]
Sure enough I get 29 methods with diffs from the fix #78074, including one that's at the top of the stack above:
```
4 (0.88 % of base) : 6775.dasm - Microsoft.CodeAnalysis.CSharp.CSharpCompilation:GetUnaliasedReferencedAssemblies(Microsoft.CodeAnalysis.PooledObjects.ArrayBuilder`1[Microsoft.CodeAnalysis.CSharp.Symbols.AssemblySymbol]):this
```
So very likely this was the same bug as #77661.

**Comment by AndyAyersMS** [comment]
Full set of impacted methods, in case it's of interest:
```
Top method regressions (bytes):
           8 (0.27 % of base) : 28193.dasm - <CreateIndexAsync>d__8:MoveNext():this
           8 (4.08 % of base) : 30113.dasm - Microsoft.CodeAnalysis.CSharp.CodeGen.CodeGenerator:EmitStatementList(Microsoft.CodeAnalysis.CSharp.BoundStatementList):this
           8 (2.13 % of base) : 6958.dasm - Microsoft.CodeAnalysis.CSharp.CSharpCompilation:GetAllUnaliasedModules(Microsoft.CodeAnalysis.PooledObjects.ArrayBuilder`1[Microsoft.CodeAnalysis.CSharp.Symbols.ModuleSymbol]):this
           8 (0.11 % of base) : 19956.dasm - Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel:GetSemanticSymbols(Microsoft.CodeAnalysis.CSharp.BoundExpression,Microsoft.CodeAnalysis.CSharp.BoundNode,Microsoft.CodeAnalysis.CSharp.Binder,int,byref,byref,byref):System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.CSharp.Symbol]:this
           8 (0.66 % of base) : 18983.dasm - Microsoft.CodeAnalysis.CSharp.InitializerRewriter:RewriteInitializersAsStatements(Microsoft.CodeAnalysis.CSharp.BoundInitializer):Microsoft.CodeAnalysis.CSharp.BoundStatement
           8 (1.39 % of base) : 6569.dasm - Microsoft.CodeAnalysis.CSharp.MergedTypeDeclaration:get_NameLocations():System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.SourceLocation]:this
           8 (0.58 % of base) : 16007.dasm - Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver`1[ushort]:TryExecuteDeclaringReferenceActions(Microsoft.CodeAnalysis.SyntaxReference,int,Microsoft.CodeAnalysis.Diagnostics.SymbolDeclaredCompilationEvent,Microsoft.CodeAnalysis.Diagnostics.AnalysisScope,Microsoft.CodeAnalysis.Diagnostics.AnalysisState,GroupedAnalyzerActions[ushort],GroupedAnalyzerActions[ushort],bool,bool,bool,bool,bool,System.Threading.CancellationToken):bool:this
           8 (2.53 % of base) : 24143.dasm - Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles.SymbolSpecification:GetHashCode():int:this
           4 (0.67 % of base) : 16216.dasm - <GetAttributes>d__5:MoveNext():bool:this
           4 (0.85 % of base) : 4101.dasm - Cache:get_ReferenceDirectives():System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.ReferenceDirective]:this
           4 (0.29 % of base) : 26398.dasm - Microsoft.CodeAnalysis.CSharp.AbstractFlowPass`2[LocalState,System.__Canon]:VisitTryStatement(Microsoft.CodeAnalysis.CSharp.BoundTryStatement):Microsoft.CodeAnalysis.CSharp.BoundNode:this
           4 (0.11 % of base) : 23014.dasm - Microsoft.CodeAnalysis.CSharp.Binder:GenerateImplicitConversionError(Microsoft.CodeAnalysis.CSharp.BindingDiagnosticBag,Microsoft.CodeAnalysis.SyntaxNode,Microsoft.CodeAnalysis.CSharp.Conversion,Microsoft.CodeAnalysis.CSharp.BoundExpression,Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol):this
           4 (6.67 % of base) : 23017.dasm - Microsoft.CodeAnalysis.CSharp.BoundTreeWalker:VisitBadExpression(Microsoft.CodeAnalysis.CSharp.BoundBadExpression):Microsoft.CodeAnalysis.CSharp.BoundNode:this
           4 (0.34 % of base) : 30422.dasm - Microsoft.CodeAnalysis.CSharp.CodeGen.CodeGenerator:TryEmitAssignmentInPlace(Microsoft.CodeAnalysis.CSharp.BoundAssignmentOperator,bool):bool:this
           4 (0.88 % of base) : 6775.dasm - Microsoft.CodeAnalysis.CSharp.CSharpCompilation:GetUnaliasedReferencedAssemblies(Microsoft.CodeAnalysis.PooledObjects.ArrayBuilder`1[Microsoft.CodeAnalysis.CSharp.Symbols.AssemblySymbol]):this
           4 (0.90 % of base) : 27398.dasm - Microsoft.CodeAnalysis.CSharp.Symbols.SourceAssemblySymbol:MightContainNoPiaLocalTypes():bool:this
           4 (2.27 % of base) : 14276.dasm - Microsoft.CodeAnalysis.CSharp.Symbols.SourceTypeParameterSymbol:get_HasValueTypeConstraint():bool:this
           4 (5.00 % of base) : 26438.dasm - Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax.DirectiveStack:GetPreviousIf(Roslyn.Utilities.ConsList`1[Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax.Directive]):Roslyn.Utilities.ConsList`1[Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax.Directive]
           4 (1.06 % of base) : 13896.dasm - Microsoft.CodeAnalysis.Diagnostics.HostSessionStartAnalysisScope:RegisterCompilationStartAction(Microsoft.CodeAnalysis.Diagnostics.DiagnosticAnalyzer,System.Action`1[Microsoft.CodeAnalysis.Diagnostics.CompilationStartAnalysisContext]):this
           4 (1.43 % of base) : 10665.dasm - Microsoft.CodeAnalysis.NodeStateTable`1[System.Collections.Immutable.ImmutableArray`1[System.__Canon]]:Single():System.ValueTuple`2[System.Collections.Immutable.ImmutableArray`1[System.__Canon],System.__Canon]:this
           4 (0.97 % of base) : 29525.dasm - Microsoft.CodeAnalysis.Operations.VariableDeclarationOperation:GetCurrent(int,int):Microsoft.CodeAnalysis.IOperation:this
           4 (0.72 % of base) : 5692.dasm - Microsoft.VisualStudio.Composition.ImportDefinitionBinding:get_ImportingSiteElementTypeRef():Microsoft.VisualStudio.Composition.Reflection.TypeRef:this
           4 (1.30 % of base) : 5542.dasm - Microsoft.VisualStudio.Composition.LazyServices:IsAnyLazyType(Microsoft.VisualStudio.Composition.Reflection.TypeRef):bool
           4 (0.29 % of base) : 5680.dasm - Microsoft.VisualStudio.Composition.ReflectionHelpers:IsAssignableTo(Microsoft.VisualStudio.Composition.ImportDefinitionBinding,Microsoft.VisualStudio.Composition.ExportDefinitionBinding):int
           4 (5.26 % of base) : 2075.dasm - System.Collections.Immutable.ImmutableArray:ToImmutableArray[System.__Canon](System.Collections.Generic.IEnumerable`1[System.__Canon]):System.Collections.Immutable.ImmutableArray`1[System.__Canon]
           4 (11.11 % of base) : 9768.dasm - System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.CSharp.Conversion]:op_Inequality(System.Nullable`1[System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.CSharp.Conversion]],System.Nullable`1[System.Collections.Immutable.ImmutableArray`1[Microsoft.CodeAnalysis.CSharp.Conversion]]):bool
           4 (3.85 % of base) : 11735.dasm - System.Collections.Immutable.ImmutableArray`1[System.__Canon]:OfType[System.__Canon]():System.Collections.Generic.IEnumerable`1[System.__Canon]:this
           4 (6.67 % of base) : 27346.dasm - System.Collections.Immutable.ImmutableArray`1[System.ValueTuple`2[System.__Canon,System.__Canon]]:AddRange(System.Collections.Generic.IEnumerable`1[System.ValueTuple`2[System.__Canon,System.__Canon]]):System.Collections.Immutable.ImmutableArray`1[System.ValueTuple`2[System.__Canon,System.__Canon]]:this
           4 (1.23 % of base) : 22494.dasm - System.Collections.Immutable.ImmutableExtensions:TryCopyTo[Microsoft.CodeAnalysis.Simplification.NodeOrTokenToReduce](System.Collections.Generic.IEnumerable`1[Microsoft.CodeAnalysis.Simplification.NodeOrTokenToReduce],Microsoft.CodeAnalysis.Simplification.NodeOrTokenToReduce[],int):bool
```
